roles/oneview_server_profile_facts/README.md

```diff
@@ -0,0 +1,37 @@
+## oneview_server_profile_facts
+Manage OneView Server Profile resources.
+
+#### Synopsis
+ Provides an interface to manage Server Profile resources. Can create, update, and delete.
+
+#### Requirements (on the host that executes the module)
+  * hpeOneView >= 5.8.0
+
+#### Options
+
+| Parameter     | Required    | Default  | Choices    | Comments |
```

Comment:
Please change the README and update it with facts info
```
| Parameter     | Required    | Default  | Choices    | Comments |
| ------------- |-------------| ---------|----------- |--------- |
| config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
| name  |   |  | |  Server Profile name.  |
| options  |   |  | |  List with options to gather additional facts about Server Profile related resources. Options allowed: `schema`, `compliancePreview`, `profilePorts`, `messages`, `transformation`, `available_networks`, `available_servers`, `available_storage_system`, `available_storage_systems`, `available_targets`, `newProfileTemplate`,  To gather facts about `compliancePreview`, `messages`, `newProfileTemplate` and `transformation` a Server Profile name is required. Otherwise, these options will be ignored.  |
| params  |   No  |  | |  List of params to delimit, filter and sort the list of resources.  params allowed: `start`: The first item to return, using 0-based indexing. `count`: The number of resources to return. `filter`: A general filter/query string to narrow the list of items returned. `sort`: The sort order of the returned data set.  |
| uri  |   |  | |  Server Profile uri.  |
```

Reply:
done